ee/app/livechat-enterprise/server/lib/Department.js

```diff
@@ -30,3 +31,23 @@ export const findAllDepartmentsByUnit = async (unitId, offset, count) => {
 
 	return { departments, total };
 };
+
+export const findAllDepartmentsByTag = async (tagId, offset, count) => {
+	const tag = LivechatTag.findOneById(tagId);
+
+	let departmentsResult = [];
+	let total = 0;
+
+	if (!tag) {
+		return { departmentsResult };
```

Comment:
Is expected that when no tags are provided, it returns `{ departmentsResult: [] }` but with tags the key is `departments`?